addons/website/models/website_visitor.py

```diff
@@ -42,8 +42,8 @@ class WebsiteVisitor(models.Model):
     country_flag = fields.Binary(related="country_id.image", string="Country Flag")
     lang_id = fields.Many2one('res.lang', string='Language', help="Language from the website when visitor has been created")
     timezone = fields.Selection(_tz_get, string='Timezone')
-    email = fields.Char(string='Email', compute='_compute_email_phone')
-    mobile = fields.Char(string='Mobile Phone', compute='_compute_email_phone')
+    email = fields.Char(string='Email', compute='_compute_email_phone', store=True)
+    mobile = fields.Char(string='Mobile Phone', compute='_compute_email_phone', store=True)
```

Comment:
Why do they need to be stored ?

Reply:
@dbeguin not 100%, but I think it's because of this line
https://github.com/odoo/odoo/blob/13.0/addons/mass_mailing/models/mailing.py#L416

We perform a SQL request to get the email value

I'll check in the morning when I arrive that it's about that.

Reply:
Ah yes, It could be, indeed !
Can you add the explanation of why (because why is more important than how) in your commit message ? Thanks !

Reply:
@dbeguin done :)